osquery/tables/networking/darwin/wifi_status.mm

```diff
@@ -34,10 +65,9 @@ QueryData genWifiStatus(QueryContext& context) {
       if (strptr != nil) {
         r["bssid"] = std::string([strptr UTF8String]);
       }
-      strptr = [interface ssid];
-      if (strptr != nil) {
-        r["network_name"] = std::string([strptr UTF8String]);
-      }
+      r["network_name"] =
+          std::string([[wifiNetworks objectForKey:[interface interfaceName]]
+                           ?: @"" UTF8String]);
```

Comment:
Get the network name from the map we created, defaulting to "" if for some reason we don't have a map entry for it.